### PR TITLE
Tr translation

### PR DESCRIPTION
**Pull request by Apsimati**
- [ Y ] I did these translations myself and own copyright for them
- [ Y] I didn't use a machine to do these translations
- [ Y] I assign all copyright to Loom Network for these translations

I'll continue translation contribution...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# CryptoZombies - Content Localization
+# CryptoZombies - İçerik Yerelleştirme
 
-This repository is used to translate CryptoZombies to other languages.
```
Comment (mattkanwisher) [inline README.md]: Please remove the README.md changes

File: tr/lander.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+en:
+  ogdescription: CryptoZombies is an interactive code school that teaches you to write smart contracts in Solidity through building your own crypto-collectables game.
```
Comment (mattkanwisher) [inline tr/lander.yml]: lander.yml needs to be translated into Turkish

File: tr/share-template-msgs.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "1": {
```
Comment (mattkanwisher) [inline tr/share-template-msgs.json]: share-template-msgs.jsonl needs to be translated into Turkish

**Comment by mattkanwisher** [comment]
Great work, lets remove English only files. for now they will have to use the English chapters for those. The Lander.yml and share-msg file are also required before launching a new language

**Comment by Apsimati** [comment]
README.md turned to its original form and required files translated to Turkish language

**Comment by mattkanwisher** [comment]
Please also remove all English language versions of the files

**Comment by Apsimati** [comment]
All English language versions of the files removed

File: tr/index.json [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+            "tryItOut": "Rastgele zombi oluşturucu deneyin.",
+            "below": "aşağı",
+            "above": "yukarı", 
+            "zombieGeneratorText": "{tryItOut} Yeni bir zombi oluşturmak için kutuya {below}{above} bir isim girin !",
+            "nameInputPlaceholder": "Bir zombi oluşturmak için bir isim girin",
+            "callToAction": [
+                "Kendi CryptoZombie'nizi yapmak ve sıralamaya katılmak istiyor musunuz?",
+                "CryptoZombilerle ÜCRETSİZ Ethereum'da kendi oyunlarınızı yapmayı öğrenin! Şimdi Başla:"
+            ]
+        },Bir
```
Comment (enlight) [inline tr/index.json]: `Bir` looks like a typo

File: tr/lander.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+en:
```
Comment (enlight) [inline tr/lander.yml]: `en` -> `tr`

**Comment by Apsimati** [comment]
Fixed

**Comment by Apsimati** [comment]
I will be working on lesson files

**Comment by Apsimati** [comment]
Lesson 1 translated into Turkish (completed)

**Comment by Apsimati** [comment]
It's all right now.

**Pull request merged**